src/js/internal/sql/shared.ts

```diff
@@ -231,6 +232,19 @@ function defaultToPostgresIfNoProtocol(url: string | URL | null): URL {
   }
   return new URL("postgres://" + url);
 }
+function adapterFromEnv(env: Bun.Env): Bun.SQL.__internal.Adapter {
+  // postgres url or postgres host provided
```

Comment:
I assume it's intentional there's no logic for SQLite here, we should add a comment explaining why? I can't tell just from reading the code here

Otherwise if it's not intentional, we should still support DATABASE_URL, for example, for sqlite